model/indexes.go

```diff
@@ -260,7 +260,7 @@ func GetRequiredIndexes() []SystemIndexes {
 	}
 }
 
-// CheckIndexes checks that all the given indexes are in the database. It does
+// CheckIndexes checks that all the given indexes are. It does
```

Comment:
This should still have `in the DB` or it can just say `all the given indexes exist.` Whichever you prefer.

Reply:
I did the former.